app/src/main/res/layout/download_item.xml

```diff
@@ -18,31 +18,34 @@
     android:adjustViewBounds="true"
     android:minHeight="?android:attr/listPreferredItemHeight"
     android:scaleType="fitCenter"
-    android:src="@mipmap/kiwix_icon" />
+    android:src="@mipmap/kiwix_icon"
+      tools:ignore="ContentDescription,RtlHardcoded" />
 
   <LinearLayout
     android:layout_width="@dimen/dimen_0dp"
     android:layout_height="wrap_content"
     android:layout_weight="1"
     android:gravity="center"
     android:orientation="vertical"
-    tools:ignore="HardcodedText">
+    tools:ignore="HardcodedText,InefficientWeight">
 
     <TextView
       android:id="@+id/title"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:layout_gravity="left"
       android:text="Title"
-      android:textAppearance="?android:attr/textAppearanceListItem" />
+      android:textAppearance="?android:attr/textAppearanceListItem"
+        tools:ignore="RtlHardcoded" />
 
     <TextView
       android:id="@+id/description"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:layout_gravity="left"
       android:text="Description"
-      android:textAppearance="?android:attr/textAppearanceSmall" />
+      android:textAppearance="?android:attr/textAppearanceSmall"
+        tools:ignore="RtlHardcoded" />
```

Comment:
Similarly here, you need to remove the hardcoded text instead of suppressing the warning.